docs/en_us/internal/testing.rst

```diff
@@ -610,6 +616,24 @@ During acceptance test execution, Django log files are written to
 
 **Note**: The acceptance tests can *not* currently run in parallel.
 
+
+Testing internationalization with dummy translations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Any text you add to the platform should be internationalized. To generate
+translations for your new strings, run the following:
+
+::
+
+    paver i18n_dummy
+
+This will generate dummy translations for the individual language files.
```

Comment:
I can't tell if this info is covered elsewhere, but what are "the individual language files" you refer to? Does this mean: "This command generates dummy translations for all strings are in X files" or "This command generates dummy translations for each language supported in the platform and puts the dummy strings in the appropriate language files". Suggest also providing more details about the dummy translation output: file type, names, location.